src/Visitor/Usage/FindLanguageDeprecations.php

```diff
@@ -33,9 +33,11 @@ public function setPhpFileInfo(PhpFileInfo $phpFileInfo)
     public function enterNode(Node $node)
     {
         if ($node instanceof Node\Expr\AssignRef) {
-            $this->phpFileInfo->addDeprecatedLanguageUsage(
-                new DeprecatedLanguageUsage('assign by reference(&=)', 'Since PHP 5.3 use normal assignment instead.', $node->getLine())
-            );
+            if ($node->expr instanceof Node\Expr\New_) {
```

Comment:
First, I change the code with "combine" style what you said.
And then, I think if use "combine" style, some cases need to change this logic, more lines need to update compare to "nest" style. So I change it to "combine" style.

Finally, I will change the doc follow you said "combine" style.